sklearn/cluster/_bicluster.py

```diff
@@ -104,14 +106,17 @@ def __init__(
         self.n_init = n_init
         self.random_state = random_state
 
-    def _check_parameters(self):
+    def _check_parameters(self, n_samples):
         legal_svd_methods = ("randomized", "arpack")
         if self.svd_method not in legal_svd_methods:
             raise ValueError(
                 "Unknown SVD method: '{0}'. svd_method must be one of {1}.".format(
                     self.svd_method, legal_svd_methods
                 )
             )
+        check_scalar(self.n_init, "n_init", target_type=numbers.Integral, min_val=1)
+        if hasattr(self, 'n_jobs') and self.n_jobs is not None:
+            check_scalar(self.n_jobs, "n_jobs", numbers.Integral)
```

Comment:
I do not think any subclass of `BaseSpectral` define `n_jobs`:

```suggestion
```

(`SpectralCoclustering` use to define it, but it has been removed in 1.0)